### ISSUE TITLE
Console warning on custom audio

### ISSUE THREAD

**Issue by gregorskii** [kosmetism/react-soundplayer#94]
Hi,

I believe the custom audio component does not require a soundcloud api token. The console complains when using `withCustomAudio`.

Output:

```
SoundCloud API requires clientId or custom apiUrl
```

Is it possible to use this player to play basic audio without soundcloud?

**Comment by voronianski** [comment]
@gregorskii yes, it's absolutely possible to use the player without SoundCloud `clientId`. Nevermind the warning so far. There's a plan to get rid of it.

**Comment by sbrichardson** [comment]
You can pass a random string to silence it. `clientId="xxx"`

**Comment by devpascoe** [comment]
I don't think that works anymore. When i do i get "Uncaught (in promise) DOMException: The element has no supported sources." in console when i click play.

**Comment by Handsoff-studio** [comment]
> I don't think that works anymore. When i do i get "Uncaught (in promise) DOMException: The element has no supported sources." in console when i click play.

Are you able to resolve the issue? I am encountering the same problem.
